BrowserKit/Sources/ToolbarKit/AddressToolbar/BrowserAddressToolbar.swift

```diff
@@ -180,10 +195,32 @@ public class BrowserAddressToolbar: UIView, AddressToolbar, ThemeApplicable, Loc
         setupAccessibility()
     }
 
+    private func setupObservers() {
+        NotificationCenter.default.addObserver(
```

Comment:
Let's use the `Notifiable` protocol here instead.

Reply:
We need to import `Shared` module to be able to use `Notifiable` protocol. Is it worth it?

Reply:
I don't think `Shared` can be imported in BrowserKit. We'd have to move `Notifiable` to Common. I don't think it's worth it for now.